Freetype 2.9.1 release is broken for Windows CMake build. 
It was fixed two commits later but not re-released. 
This patch cherry-picks the fix from http://git.savannah.gnu.org/cgit/freetype/freetype2.git/commit/?id=660afb5
Next time Freetype is updated, remove this from build.win*/third-party/Makefile.freetype2

Upstream bug report https://savannah.nongnu.org/bugs/?func=detailitem&item_id=54846 
says "CMake fails to install on Windows - Don't worry, it's not your fault. I broke 
it myself and found out after the release, but didn't bug Werner to release a 2.9.1.1"

Our discussion here https://github.com/OpenSmalltalk/opensmalltalk-vm/issues/319


diff -ur freetype-2.9.1/CMakeLists.txt freetype-2.9.1.fixed/CMakeLists.txt
--- freetype-2.9.1/CMakeLists.txt	2018-05-01 18:45:45.000000000 +0800
+++ freetype-2.9.1.fixed/CMakeLists.txt	2018-12-14 18:13:46.624962900 +0800
@@ -229,9 +229,12 @@
   endif ()
   string(REPLACE "/undef " "#undef "
     FTCONFIG_H "${FTCONFIG_H}")
-  file(WRITE "${PROJECT_BINARY_DIR}/include/freetype/config/ftconfig.h"
-    "${FTCONFIG_H}")
+else()
+  file(READ "${PROJECT_SOURCE_DIR}/include/freetype/config/ftconfig.h"
+    FTCONFIG_H)
 endif ()
+file(WRITE "${PROJECT_BINARY_DIR}/include/freetype/config/ftconfig.h"
+  "${FTCONFIG_H}")

